surrealdb/Client.py

```diff
@@ -0,0 +1,23 @@
+import urllib3
+import base64
+import json
+import time
+import sys
+
+class client:

```

Comment:
I have a couple of issues with this.

1. Why is the class name lowercase?
2. Why is nothing typed?
3. Why are you using urllib instead of something like aiohttp?
4. Why are there no spaces between the methods?
5. Why is this not imported in `__init__.py`?
6. Why is nothing documented?
7. Why are you returning `str`s when you could return nothing?
8. Why not make separate variables for `host`, `auth_header`, `ns` and `db`?

That's all that has immediately come to mind, I'm sure that the other reviewers will point out more.

Reply:
I see, I never thought of this stuff as Im quite new to making packages, and really never thought of this. 
Im always learning though, thanks for the feedback!